_data/developer.yml

```diff
@@ -224,6 +230,7 @@ websites:
     - name: Papertrail
       url: https://papertrailapp.com/
       twitter: papertrailapp
+      facebook: papertrail.io
```

Comment:
It looks like papertrailapp.com and papertrail.io might be separate companies. Their websites seem quite different. They also have different logos.